python/pyspark/mllib/tests.py

```diff
@@ -252,6 +252,16 @@ def test_sparse_vector_indexing(self):
         for ind in [7.8, '1']:
             self.assertRaises(TypeError, sv.__getitem__, ind)
 
+        zeros = SparseVector(4, {})
```

Comment:
Well, It should be. I played with different implementations and I just wanted to be sure I didn't break anything. I can remove this if you want.